be/test/storage/tablet_meta_manager_test.cpp

```diff
@@ -23,25 +23,25 @@
 
 namespace starrocks {
 
-namespace fs = std::filesystem;
+namespace sfs = std::filesystem;
```

Comment:
why change this? there is a conflict?

Reply:
Yes, the previous definition is in `fs_util.h` with defining `namespace starrocks::fs`. These two files had no correlation before, but this is also a potential risk.